services/learn-card-network/brain-service/src/cache/invites.ts

```diff
@@ -4,29 +4,75 @@ export const getInviteCacheKey = (profileId: string, challenge: string): string
     `inviteChallenge:${profileId}:${challenge}`;
 
 export const VALID = 'valid';
+export const NEVER_EXPIRE = 'never';
```

Comment:
Just curious, why add another value here for never? Why not just have it say "valid" and never expire? Not a big deal, just seems a tiny bit counterintuitive to have the value "never" mean that it's valid!

Reply:
@TaylorBeeston I was having trouble getting the tests to pass and setting up those `valid` and `never` values helped me avoid some implicit behavior that I was seeing.